docs/vm-installation.md

```diff
@@ -0,0 +1,99 @@
+# Installing antrea-agent on VM
+
+`antrea-agent` can be run on Linux/Windows VMs. Below are the steps needed to configure and run
+antrea-agent on VMs
+
+## Prerequisites on Kubernetes cluster
+
+1. Create a Namespace for `antrea-agent`
+2. Create a ServiceAccount, ClusterRole and ClusterRoleBinding for `antrea-agent`.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/feature/externalnode/build/yamls/externalnode/vm-agent-rbac.yaml
```

Comment:
We should add the command to create the Namespace?

As commented earlier, I suggest to use "vm-ns" as the default Namespace in the yamls, but we should still add some words to explain you need to update the yamls to use another Namespace.

Reply:
Sure, I will add command to create NameSpace. We will be changing to 'vm-ns' in build/yamls/externalnode/vm-agent-rbac.yml